lang/nl/lang.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+/**
+ * @license    GPL 2 (http://www.gnu.org/licenses/gpl.html)
+ *
+ * @author G. Uitslag <klapinklapin@gmail.com>
+ */
+$lang['template']              = '— Template';
+$lang['not_found']             = ' niet gevonden.';
```

Comment:
Hi,
are extra spaces before the translated string needed? They are difficult to spot for who make translations, and probably html parsing ignores also some of them?

Reply:
Yes.